subprojects/kotlin-dsl-tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/KotlinBuildScriptModelBuilder.kt

```diff
@@ -290,7 +290,7 @@ fun settingsScriptPluginModelBuilder(scriptFile: File, project: ProjectInternal)
     val (scriptHandler, scriptClassPath) = compilationClassPathForScriptPluginOf(
         target = settings,
         scriptFile = scriptFile,
-        baseScope = settings.rootClassLoaderScope,
+        baseScope = settings.baseClassLoaderScope,
```

Comment:
:x: this should remain `rootClassLoaderScope` for "Settings script plugins"

Reply:
That property has been renamed to baseClassLoaderScope.